docs/guides/tgi.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: "High performance model serving with TGI"
+description: "A guide to using TGI for your model"
+---
+
+TGI, or Text Generation Inference, is a high performance model server for language models built by Huggingface. In this doc, we'll cover when to use TGI, how to tune TGI for performance and how to deploy it to Baseten.
```

Comment:
consider linking back to TGI on huggingface (https://huggingface.co/text-generation-inference) here